docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/gitops/calico_k8sconfig_gitops.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# <--- Change the following environment variables according to your Azure service principal name --->
+
+echo "Exporting environment variables"
+export appId='<Your Azure service principal name>'
+export password='<Your Azure service principal password>'
+export tenantId='<Your Azure tenant ID>'
+export resourceGroup='<Azure resource group name>'
+export arcClusterName='<The name of your k8s cluster as it will be shown in Azure Arc>'
+export appClonedRepo='<The URL for the "Tigera Azure-arc-demo" cloned GitHub repository>'
+
+
+# Installing Helm 3
+echo "Installing Helm 3"
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+
+# Installing Azure CLI & Azure Arc Extensions
```

Comment:
Can we update this to simple installation?
curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
Reference: https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/cluster_api/gitops/basic/az_k8sconfig_capi.sh

Reply:
Hi Zaid, the difficulty of making it as a simple installation as we will need to change the environment variables according to the Azure service principal name, also need to update the URL after cloned GitHub repository.
I don't know how to achieve that with one curl script, open to any suggestion.

Reply:
Hi Jessie,

I meant not the whole script, just the Azure CLI installation part instead of these line of code:
sudo apt-get update
sudo apt-get install -y ca-certificates curl apt-transport-https lsb-release gnupg
curl -sL https://packages.microsoft.com/keys/microsoft.asc |
gpg --dearmor |
sudo tee /etc/apt/trusted.gpg.d/microsoft.asc.gpg > /dev/null
AZ_REPO=$(lsb_release -cs)
echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" |
sudo tee /etc/apt/sources.list.d/azure-cli.list
sudo apt-get update
sudo apt-get install azure-cli

We can replace it with single line:
curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash

Reference: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt#option-1-install-with-one-command

Reply:
Now I understand what you mean :) 
Done